ivy_tests/test_ivy/test_frontends/test_jax/test_jax_random.py

```diff
@@ -1191,3 +1191,60 @@ def call():
         assert ret_np.shape == shape
     else:
         assert ret_np.shape == b.shape
+        
+        
+        @pytest.mark.xfail
```

Comment:
This will be removed as we used `test_values=False` the tests should be able to pass